scripts/create-pr.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+PR_TITLE="chore (automated): OpenSauced updates"
+BRANCH_NAME="chore_automated_open-sauced-updates_$(date +%s)"
+
+git branch $BRANCH_NAME
+git switch $BRANCH_NAME
+
+# There are potentially multiple files if the blog post has images.
```

Comment:
> if the blog post has images.

Copy pasta?

Reply:
That is copy pasta from my own workflow. 😅